packages/components-native/src/ProgressBar/ProgressBar.tsx

```diff
@@ -34,7 +35,12 @@ export function ProgressBar({
           inProgress={inProgress}
         />
       ) : (
-        <View style={styles.progressBarContainer}>
+        <View
+          style={[
+            styles.progressBarContainer,
+            size === "small" && { height: 10 },
```

Comment:
We should reference the same sizes as web:
![image](https://github.com/user-attachments/assets/f39db7c9-fe6d-4a28-8dd3-ef151b017f4b)

```suggestion
            size === "small" && { height: 8 },
```